### PR TITLE
Log sv.max_edicts value to demos and fix typos in ServerDLL

### PR DESCRIPTION
**Pull request by hobokenn**

### CODE REVIEW

File: BunnymodXT/runtime_data.hpp [hunk 1]
```diff
@@ -48,16 +48,21 @@ namespace RuntimeData
...
-	                            CVarValues,
-	                            Time,
-	                            BoundCommand,
-	                            AliasExpansion,
-	                            ScriptExecution,
-	                            CommandExecution,
-	                            GameEndMarker,
-	                            LoadedModules,
-				    CustomTriggerCommand>;
+								CVarValues,
```
Comment (YaLTeR) [inline BunnymodXT/runtime_data.hpp:56]: no, alignment is done using spaces, that way it doesn't break if you change tab size (as happened here):

image

Comment (hobokenn) [reply BunnymodXT/runtime_data.hpp]: like this?

Comment (YaLTeR) [reply BunnymodXT/runtime_data.hpp]: Yeah

Comment (hobokenn) [reply BunnymodXT/runtime_data.hpp]: cool, ty 👍

File: BunnymodXT/modules/HwDLL.cpp [hunk 2]
```diff
@@ -4067,6 +4070,10 @@ void HwDLL::SaveInitialDataToDemo()
 
 	RuntimeData::Add(std::move(cvar_values));
 
+	const int maxEdicts = *reinterpret_cast<int *>(reinterpret_cast<uintptr_t>(sv) + offMaxEdicts);
```
Comment (YaLTeR) [inline BunnymodXT/modules/HwDLL.cpp:4073]: Probably shouldn't do that if either the pointer or the offset is missing.

**Comment by YaLTeR** [comment]
Thanks!

**Pull request merged**